doc/docs/Python_User_Interface.md

```diff
@@ -1158,7337 +1335,6462 @@ If `match_frequency=False` or `kpoint` is not zero in the given `direction`, the
+dimensions for both the cell and the force regions with the same number of
+processors.
+
+</div>
 
 </div>
 
-<a id="Simulation.get_sfield"></a>
+<a id="Simulation.load_minus_force"></a>
 
-**Simulation.get\_sfield**
+<div class="class_members" markdown="1">
 
 ```python
-def get_sfield(self):
+def load_minus_force(self, fname, force):
 ```
 
 <div class="method_docstring" markdown="1">
 
+As `load_force`, but negates the Fourier-transformed fields after they are loaded.
+This means that they will be *subtracted* from any future field Fourier transforms
+that are accumulated.
 
+</div>
 
 </div>
 
-<a id="Simulation.get_sfield_p"></a>
+To keep the fields in memory and avoid writing to and reading from a file, use the following three `Simulation` methods:
 
-**Simulation.get\_sfield\_p**
+
+<a id="Simulation.get_force_data"></a>
+
+<div class="class_members" markdown="1">
 
 ```python
-def get_sfield_p(self):
+def get_force_data(self, force):
 ```
 
 <div class="method_docstring" markdown="1">
 
+Get the Fourier-transformed fields corresponding to the given force object as a
+`ForceData`, which is just a named tuple of NumPy arrays. Note that this object is
+only useful for passing to `load_force_data` below and should be considered
+opaque.
 
+</div>
 
 </div>
 
-<a id="Simulation.get_sfield_r"></a>
+<a id="Simulation.load_force_data"></a>
 
-**Simulation.get\_sfield\_r**
+<div class="class_members" markdown="1">
 
 ```python
-def get_sfield_r(self):
+def load_force_data(self, force, fdata):
 ```
 
 <div class="method_docstring" markdown="1">
 
+Load the Fourier-transformed fields into the given force object (replacing any
+values currently there) from the `ForceData` object `fdata`. You must load from an
+object that was created by `get_force_data` in a simulation of the same dimensions
+(for both the cell and the flux regions) with the same number of processors.
 
+</div>
 
 </div>
 
-<a id="Simulation.get_sfield_x"></a>
+<a id="Simulation.load_minus_force_data"></a>
 
-**Simulation.get\_sfield\_x**
+<div class="class_members" markdown="1">
 
 ```python
-def get_sfield_x(self):
+def load_minus_force_data(self, force, fdata):
 ```
 
 <div class="method_docstring" markdown="1">
 
+As `load_force_data`, but negates the Fourier-transformed fields after they are
+loaded. This means that they will be *subtracted* from any future field Fourier
+transforms that are accumulated.
 
+</div>
 
 </div>
 
-<a id="Simulation.get_sfield_y"></a>
+### LDOS spectra
+
+Meep can also calculate the LDOS (local density of states) spectrum, as described in [Tutorial/Local Density of States](Python_Tutorials/Local_Density_of_States.md). To do this, you simply pass the following step function to your `run` command:
+
 
-**Simulation.get\_sfield\_y**
+<a id="Ldos"></a>
 
 ```python
-def get_sfield_y(self):
+def Ldos(*args):
 ```
 
-<div class="method_docstring" markdown="1">
+<div class="function_docstring" markdown="1">
 
+Ldos(fcen, df, nfreq, freq)
```

Comment:
Again: `Ldos(fcen, df, nfreq, freq)` which lists the arguments should be a code block?